{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Set up links to demux samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "mv: cannot stat ‘/home/jovyan/data/KNP_demux/r?.*.fq.gz’: No such file or directory\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mkdir -p ~/data/fastqs\n",
    "mv ~/data/KNP_demux/r?.*.fq.gz ~/data/fastqs\n",
    "rm -f ~/data/fastqs/*unmatched.fq.gz\n",
    "mv ~/data/EG_demux/r?.*.fq.gz ~/data/fastqs\n",
    "rm -f ~/data/fastqs/*unmatched.fq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "r1.7A_E.fq.gz  r1.7C_K.fq.gz  r1.8B_P.fq.gz  r2.7B_G.fq.gz  r2.8A_N.fq.gz\r\n",
      "r1.7A_G.fq.gz  r1.7C_N.fq.gz  r1.8C_E.fq.gz  r2.7B_K.fq.gz  r2.8A_P.fq.gz\r\n",
      "r1.7A_K.fq.gz  r1.7C_P.fq.gz  r1.8C_G.fq.gz  r2.7B_N.fq.gz  r2.8B_E.fq.gz\r\n",
      "r1.7A_N.fq.gz  r1.8A_E.fq.gz  r1.8C_K.fq.gz  r2.7B_P.fq.gz  r2.8B_G.fq.gz\r\n",
      "r1.7A_P.fq.gz  r1.8A_G.fq.gz  r1.8C_N.fq.gz  r2.7C_E.fq.gz  r2.8B_K.fq.gz\r\n",
      "r1.7B_E.fq.gz  r1.8A_K.fq.gz  r1.8C_P.fq.gz  r2.7C_G.fq.gz  r2.8B_N.fq.gz\r\n",
      "r1.7B_G.fq.gz  r1.8A_N.fq.gz  r2.7A_E.fq.gz  r2.7C_K.fq.gz  r2.8B_P.fq.gz\r\n",
      "r1.7B_K.fq.gz  r1.8A_P.fq.gz  r2.7A_G.fq.gz  r2.7C_N.fq.gz  r2.8C_E.fq.gz\r\n",
      "r1.7B_N.fq.gz  r1.8B_E.fq.gz  r2.7A_K.fq.gz  r2.7C_P.fq.gz  r2.8C_G.fq.gz\r\n",
      "r1.7B_P.fq.gz  r1.8B_G.fq.gz  r2.7A_N.fq.gz  r2.8A_E.fq.gz  r2.8C_K.fq.gz\r\n",
      "r1.7C_E.fq.gz  r1.8B_K.fq.gz  r2.7A_P.fq.gz  r2.8A_G.fq.gz  r2.8C_N.fq.gz\r\n",
      "r1.7C_G.fq.gz  r1.8B_N.fq.gz  r2.7B_E.fq.gz  r2.8A_K.fq.gz  r2.8C_P.fq.gz\r\n"
     ]
    }
   ],
   "source": [
    "ls ~/data/fastqs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "mkdir -p genome\n",
    "wget --output-document genome/ecoli_w3110.fa.gz --no-verbose ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA_000010245.1_ASM1024v1/GCA_000010245.1_ASM1024v1_genomic.fna.gz\n",
    "wget --output-document genome/ecoli_w3110.gff.gz --no-verbose ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA_000010245.1_ASM1024v1/GCA_000010245.1_ASM1024v1_genomic.gff.gz\n",
    "\n",
    "gunzip genome/ecoli_w3110.fa.gz\n",
    "gunzip genome/ecoli_w3110.gff.gz\n",
    "\n",
    "bowtie2-build genome/ecoli_w3110.fa genome/ecoli_w3110\n",
    "\n",
    "ls genome"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###Now Everything Together!!\n",
    "We will now run all the samples, but first we need to generate an adapter file for all the samples. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Generating a full adapter file\n",
    "We still need to do the copy-and-paste part from the index primer manual, but we will do the reverse complementing automatically.  Let's do that now (if you are feeling lazy, you can use the `testrun_adapters.fasta` file in the repo directory) . . .\n",
    "\n",
    "OK, now that it is out of the way, we need to install a python library that our reverse complementing script will use"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we can run our script.  By default it only outputs the reverse complemented sequences, but with the --both option, it will also output the original sequence."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 2: /home/jovyan/bioinf_nb_ngscourse2015/revcomp.py: No such file or directory\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "~/bioinf_nb_ngscourse2015/revcomp.py \\\n",
    "    ~/bioinf_nb_ngscourse2015/all_neb_adapters.fasta \\\n",
    "    --both --output all_neb_adapters_both.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "####Looping over all the samples\n",
    "Now we can put all of the previous commands into one big loop.  This is probably a good time for copying and pasting.  But we will make a few small changes.\n",
    "\n",
    "1. We will add all the barcodes the the list of barcodes to iterate over\n",
    "    * AGTCAA AGTTCC ATGTCA CCGTCC GTCCGC GTGAAA\n",
    "2. We need to remember to use our full adapter file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "IFS=\".\"\n",
    "# for R1 in ~/scratch/analysis/all_data/r1.*.fq.gz\n",
    "for R1 in ~/scratch/analysis/all_data/r1.7A_E.fq.gz\n",
    "    do\n",
    "        set $R1\n",
    "        BARCODE=$2\n",
    "        echo $BARCODE\n",
    "    done\n",
    "unset IFS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/jovyan/work/BioInformaticsNotebooks2015\n",
      "all_neb_adapters.fasta\n",
      "annotation.csv\n",
      "annotation.ipynb\n",
      "annotation_prep.ipynb\n",
      "check_R.R\n",
      "counting.ipynb\n",
      "demux_exploration.ipynb\n",
      "DESeq2-Notebook-from-Matrix.ipynb\n",
      "DESeq2-Notebook.ipynb\n",
      "file_transfer_and_visualization.ipynb\n",
      "indices.csv\n",
      "install_fastq_multx.sh\n",
      "mapping.ipynb\n",
      "neb_19_adapter.fasta\n",
      "parse_gtf.py\n",
      "pool1_barcodes.tab\n",
      "pool2_barcodes.tab\n",
      "preprocessing.ipynb\n",
      "putting_it_together_all_data-Create_OldCounts.ipynb\n",
      "putting_it_together_all_data.ipynb\n",
      "putting_it_together.ipynb\n",
      "revcomp.py\n",
      "testrun_adapters.fasta\n",
      "test_run_counts_OLD\n",
      "update_R_notes.txt\n",
      "update_R.sh\n",
      "working_with_your_data-Copy1.ipynb\n",
      "working_with_your_data.ipynb\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "pwd\n",
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"./revcomp.py\", line 33, in <module>\n",
      "    main()\n",
      "  File \"./revcomp.py\", line 11, in main\n",
      "    parser.add_argument(\"FASTA_FILE\", type=file,\n",
      "NameError: name 'file' is not defined\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "./revcomp.py all_neb_adapters.fasta \\\n",
    "    --both --output all_neb_adapters_both.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "7A_E\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7A_E.fq.gz /home/jovyan/data/fastqs/r2.7A_E.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7A_E.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7A_E.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1409305\n",
      "Too short after clip: 2\n",
      "Trimmed 111072 reads (/home/jovyan/data/fastqs/r1.7A_E.fq.gz) by an average of 1.46 bases on quality < 20\n",
      "Trimmed 185904 reads (/home/jovyan/data/fastqs/r2.7A_E.fq.gz) by an average of 1.70 bases on quality < 20\n",
      "7A_G\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7A_G.fq.gz /home/jovyan/data/fastqs/r2.7A_G.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7A_G.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7A_G.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1441438\n",
      "Too short after clip: 12\n",
      "Trimmed 108618 reads (/home/jovyan/data/fastqs/r1.7A_G.fq.gz) by an average of 1.48 bases on quality < 20\n",
      "Trimmed 174550 reads (/home/jovyan/data/fastqs/r2.7A_G.fq.gz) by an average of 1.78 bases on quality < 20\n",
      "7A_K\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7A_K.fq.gz /home/jovyan/data/fastqs/r2.7A_K.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7A_K.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7A_K.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1409221\n",
      "Too short after clip: 49\n",
      "Trimmed 98025 reads (/home/jovyan/data/fastqs/r1.7A_K.fq.gz) by an average of 1.46 bases on quality < 20\n",
      "Trimmed 173794 reads (/home/jovyan/data/fastqs/r2.7A_K.fq.gz) by an average of 1.73 bases on quality < 20\n",
      "7A_N\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7A_N.fq.gz /home/jovyan/data/fastqs/r2.7A_N.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7A_N.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7A_N.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1732143\n",
      "Too short after clip: 36\n",
      "Trimmed 113298 reads (/home/jovyan/data/fastqs/r1.7A_N.fq.gz) by an average of 1.41 bases on quality < 20\n",
      "Trimmed 192279 reads (/home/jovyan/data/fastqs/r2.7A_N.fq.gz) by an average of 1.75 bases on quality < 20\n",
      "7A_P\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7A_P.fq.gz /home/jovyan/data/fastqs/r2.7A_P.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7A_P.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7A_P.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1377416\n",
      "Too short after clip: 29\n",
      "Trimmed 95472 reads (/home/jovyan/data/fastqs/r1.7A_P.fq.gz) by an average of 1.45 bases on quality < 20\n",
      "Trimmed 182266 reads (/home/jovyan/data/fastqs/r2.7A_P.fq.gz) by an average of 1.73 bases on quality < 20\n",
      "7B_E\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7B_E.fq.gz /home/jovyan/data/fastqs/r2.7B_E.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7B_E.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7B_E.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1473906\n",
      "Too short after clip: 18\n",
      "Trimmed 111369 reads (/home/jovyan/data/fastqs/r1.7B_E.fq.gz) by an average of 1.54 bases on quality < 20\n",
      "Trimmed 208336 reads (/home/jovyan/data/fastqs/r2.7B_E.fq.gz) by an average of 1.82 bases on quality < 20\n",
      "7B_G\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7B_G.fq.gz /home/jovyan/data/fastqs/r2.7B_G.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7B_G.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7B_G.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1632838\n",
      "Too short after clip: 2\n",
      "Trimmed 125961 reads (/home/jovyan/data/fastqs/r1.7B_G.fq.gz) by an average of 1.42 bases on quality < 20\n",
      "Trimmed 209330 reads (/home/jovyan/data/fastqs/r2.7B_G.fq.gz) by an average of 1.73 bases on quality < 20\n",
      "7B_K\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7B_K.fq.gz /home/jovyan/data/fastqs/r2.7B_K.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7B_K.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7B_K.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1620821\n",
      "Too short after clip: 36\n",
      "Trimmed 117638 reads (/home/jovyan/data/fastqs/r1.7B_K.fq.gz) by an average of 1.41 bases on quality < 20\n",
      "Trimmed 196270 reads (/home/jovyan/data/fastqs/r2.7B_K.fq.gz) by an average of 1.69 bases on quality < 20\n",
      "7B_N\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7B_N.fq.gz /home/jovyan/data/fastqs/r2.7B_N.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7B_N.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7B_N.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1059581\n",
      "Too short after clip: 23\n",
      "Trimmed 73086 reads (/home/jovyan/data/fastqs/r1.7B_N.fq.gz) by an average of 1.57 bases on quality < 20\n",
      "Trimmed 127298 reads (/home/jovyan/data/fastqs/r2.7B_N.fq.gz) by an average of 1.87 bases on quality < 20\n",
      "7B_P\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7B_P.fq.gz /home/jovyan/data/fastqs/r2.7B_P.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7B_P.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7B_P.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1594140\n",
      "Too short after clip: 31\n",
      "Trimmed 109487 reads (/home/jovyan/data/fastqs/r1.7B_P.fq.gz) by an average of 1.46 bases on quality < 20\n",
      "Trimmed 194037 reads (/home/jovyan/data/fastqs/r2.7B_P.fq.gz) by an average of 1.68 bases on quality < 20\n",
      "7C_E\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7C_E.fq.gz /home/jovyan/data/fastqs/r2.7C_E.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7C_E.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7C_E.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1730095\n",
      "Too short after clip: 11\n",
      "Trimmed 141184 reads (/home/jovyan/data/fastqs/r1.7C_E.fq.gz) by an average of 1.52 bases on quality < 20\n",
      "Trimmed 248840 reads (/home/jovyan/data/fastqs/r2.7C_E.fq.gz) by an average of 1.79 bases on quality < 20\n",
      "7C_G\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7C_G.fq.gz /home/jovyan/data/fastqs/r2.7C_G.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7C_G.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7C_G.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1501468\n",
      "Too short after clip: 5\n",
      "Trimmed 114746 reads (/home/jovyan/data/fastqs/r1.7C_G.fq.gz) by an average of 1.44 bases on quality < 20\n",
      "Trimmed 187905 reads (/home/jovyan/data/fastqs/r2.7C_G.fq.gz) by an average of 1.74 bases on quality < 20\n",
      "7C_K\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7C_K.fq.gz /home/jovyan/data/fastqs/r2.7C_K.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7C_K.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7C_K.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1443192\n",
      "Too short after clip: 28\n",
      "Trimmed 101554 reads (/home/jovyan/data/fastqs/r1.7C_K.fq.gz) by an average of 1.42 bases on quality < 20\n",
      "Trimmed 185088 reads (/home/jovyan/data/fastqs/r2.7C_K.fq.gz) by an average of 1.71 bases on quality < 20\n",
      "7C_N\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7C_N.fq.gz /home/jovyan/data/fastqs/r2.7C_N.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7C_N.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7C_N.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1288307\n",
      "Too short after clip: 27\n",
      "Trimmed 87115 reads (/home/jovyan/data/fastqs/r1.7C_N.fq.gz) by an average of 1.57 bases on quality < 20\n",
      "Trimmed 159346 reads (/home/jovyan/data/fastqs/r2.7C_N.fq.gz) by an average of 1.89 bases on quality < 20\n",
      "7C_P\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.7C_P.fq.gz /home/jovyan/data/fastqs/r2.7C_P.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.7C_P.trim.fq.gz -o /home/jovyan/data/trimmed/r2.7C_P.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1333891\n",
      "Too short after clip: 27\n",
      "Trimmed 91487 reads (/home/jovyan/data/fastqs/r1.7C_P.fq.gz) by an average of 1.46 bases on quality < 20\n",
      "Trimmed 168673 reads (/home/jovyan/data/fastqs/r2.7C_P.fq.gz) by an average of 1.68 bases on quality < 20\n",
      "8A_E\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8A_E.fq.gz /home/jovyan/data/fastqs/r2.8A_E.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8A_E.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8A_E.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1741353\n",
      "Too short after clip: 12\n",
      "Trimmed 132799 reads (/home/jovyan/data/fastqs/r1.8A_E.fq.gz) by an average of 1.48 bases on quality < 20\n",
      "Trimmed 220909 reads (/home/jovyan/data/fastqs/r2.8A_E.fq.gz) by an average of 1.74 bases on quality < 20\n",
      "8A_G\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8A_G.fq.gz /home/jovyan/data/fastqs/r2.8A_G.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8A_G.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8A_G.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1853498\n",
      "Too short after clip: 7\n",
      "Trimmed 135495 reads (/home/jovyan/data/fastqs/r1.8A_G.fq.gz) by an average of 1.45 bases on quality < 20\n",
      "Trimmed 241259 reads (/home/jovyan/data/fastqs/r2.8A_G.fq.gz) by an average of 1.77 bases on quality < 20\n",
      "8A_K\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8A_K.fq.gz /home/jovyan/data/fastqs/r2.8A_K.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8A_K.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8A_K.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1189847\n",
      "Too short after clip: 16\n",
      "Trimmed 84475 reads (/home/jovyan/data/fastqs/r1.8A_K.fq.gz) by an average of 1.41 bases on quality < 20\n",
      "Trimmed 130681 reads (/home/jovyan/data/fastqs/r2.8A_K.fq.gz) by an average of 1.65 bases on quality < 20\n",
      "8A_N\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8A_N.fq.gz /home/jovyan/data/fastqs/r2.8A_N.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8A_N.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8A_N.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1157256\n",
      "Too short after clip: 202\n",
      "Trimmed 87172 reads (/home/jovyan/data/fastqs/r1.8A_N.fq.gz) by an average of 2.45 bases on quality < 20\n",
      "Trimmed 153650 reads (/home/jovyan/data/fastqs/r2.8A_N.fq.gz) by an average of 2.44 bases on quality < 20\n",
      "8A_P\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8A_P.fq.gz /home/jovyan/data/fastqs/r2.8A_P.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8A_P.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8A_P.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1144671\n",
      "Too short after clip: 33\n",
      "Trimmed 77394 reads (/home/jovyan/data/fastqs/r1.8A_P.fq.gz) by an average of 1.43 bases on quality < 20\n",
      "Trimmed 147258 reads (/home/jovyan/data/fastqs/r2.8A_P.fq.gz) by an average of 1.72 bases on quality < 20\n",
      "8B_E\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8B_E.fq.gz /home/jovyan/data/fastqs/r2.8B_E.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8B_E.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8B_E.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1855951\n",
      "Too short after clip: 16\n",
      "Trimmed 140936 reads (/home/jovyan/data/fastqs/r1.8B_E.fq.gz) by an average of 1.50 bases on quality < 20\n",
      "Trimmed 243056 reads (/home/jovyan/data/fastqs/r2.8B_E.fq.gz) by an average of 1.77 bases on quality < 20\n",
      "8B_G\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8B_G.fq.gz /home/jovyan/data/fastqs/r2.8B_G.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8B_G.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8B_G.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1245813\n",
      "Too short after clip: 6\n",
      "Trimmed 95098 reads (/home/jovyan/data/fastqs/r1.8B_G.fq.gz) by an average of 1.44 bases on quality < 20\n",
      "Trimmed 184360 reads (/home/jovyan/data/fastqs/r2.8B_G.fq.gz) by an average of 1.90 bases on quality < 20\n",
      "8B_K\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8B_K.fq.gz /home/jovyan/data/fastqs/r2.8B_K.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8B_K.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8B_K.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1602559\n",
      "Too short after clip: 38\n",
      "Trimmed 111207 reads (/home/jovyan/data/fastqs/r1.8B_K.fq.gz) by an average of 1.42 bases on quality < 20\n",
      "Trimmed 195371 reads (/home/jovyan/data/fastqs/r2.8B_K.fq.gz) by an average of 1.69 bases on quality < 20\n",
      "8B_N\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8B_N.fq.gz /home/jovyan/data/fastqs/r2.8B_N.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8B_N.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8B_N.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 955329\n",
      "Too short after clip: 27\n",
      "Trimmed 80258 reads (/home/jovyan/data/fastqs/r1.8B_N.fq.gz) by an average of 2.55 bases on quality < 20\n",
      "Trimmed 123681 reads (/home/jovyan/data/fastqs/r2.8B_N.fq.gz) by an average of 2.65 bases on quality < 20\n",
      "8B_P\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8B_P.fq.gz /home/jovyan/data/fastqs/r2.8B_P.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8B_P.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8B_P.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1196469\n",
      "Too short after clip: 34\n",
      "Trimmed 79945 reads (/home/jovyan/data/fastqs/r1.8B_P.fq.gz) by an average of 1.43 bases on quality < 20\n",
      "Trimmed 145341 reads (/home/jovyan/data/fastqs/r2.8B_P.fq.gz) by an average of 1.69 bases on quality < 20\n",
      "8C_E\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8C_E.fq.gz /home/jovyan/data/fastqs/r2.8C_E.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8C_E.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8C_E.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 2073059\n",
      "Too short after clip: 0\n",
      "Trimmed 156300 reads (/home/jovyan/data/fastqs/r1.8C_E.fq.gz) by an average of 1.42 bases on quality < 20\n",
      "Trimmed 276854 reads (/home/jovyan/data/fastqs/r2.8C_E.fq.gz) by an average of 1.77 bases on quality < 20\n",
      "8C_G\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8C_G.fq.gz /home/jovyan/data/fastqs/r2.8C_G.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8C_G.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8C_G.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1332680\n",
      "Too short after clip: 1\n",
      "Trimmed 99230 reads (/home/jovyan/data/fastqs/r1.8C_G.fq.gz) by an average of 1.46 bases on quality < 20\n",
      "Trimmed 161690 reads (/home/jovyan/data/fastqs/r2.8C_G.fq.gz) by an average of 1.76 bases on quality < 20\n",
      "8C_K\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8C_K.fq.gz /home/jovyan/data/fastqs/r2.8C_K.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8C_K.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8C_K.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1303302\n",
      "Too short after clip: 239\n",
      "Trimmed 91163 reads (/home/jovyan/data/fastqs/r1.8C_K.fq.gz) by an average of 1.66 bases on quality < 20\n",
      "Trimmed 159346 reads (/home/jovyan/data/fastqs/r2.8C_K.fq.gz) by an average of 1.87 bases on quality < 20\n",
      "8C_N\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8C_N.fq.gz /home/jovyan/data/fastqs/r2.8C_N.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8C_N.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8C_N.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 992685\n",
      "Too short after clip: 21\n",
      "Trimmed 76926 reads (/home/jovyan/data/fastqs/r1.8C_N.fq.gz) by an average of 1.41 bases on quality < 20\n",
      "Trimmed 121718 reads (/home/jovyan/data/fastqs/r2.8C_N.fq.gz) by an average of 1.91 bases on quality < 20\n",
      "8C_P\n",
      "Command Line: all_neb_adapters.fasta /home/jovyan/data/fastqs/r1.8C_P.fq.gz /home/jovyan/data/fastqs/r2.8C_P.fq.gz -q 20 -x 0.5 -o /home/jovyan/data/trimmed/r1.8C_P.trim.fq.gz -o /home/jovyan/data/trimmed/r2.8C_P.trim.fq.gz\n",
      "Scale used: 2.2\n",
      "Phred: 33\n",
      "Threshold used: 751 out of 300000\n",
      "Files: 2\n",
      "Total reads: 1281318\n",
      "Too short after clip: 31\n",
      "Trimmed 86303 reads (/home/jovyan/data/fastqs/r1.8C_P.fq.gz) by an average of 1.45 bases on quality < 20\n",
      "Trimmed 160063 reads (/home/jovyan/data/fastqs/r2.8C_P.fq.gz) by an average of 1.70 bases on quality < 20\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "[2016-07-03 02:31:21] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 02:31:21] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 02:31:21] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 02:31:21] Checking for reference FASTA file\n",
      "[2016-07-03 02:31:21] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 02:31:22] Reading known junctions from GTF file\n",
      "[2016-07-03 02:31:22] Preparing reads\n",
      "\t left reads: min. length=46, max. length=101, 1409223 kept reads (80 discarded)\n",
      "\tright reads: min. length=39, max. length=101, 1409169 kept reads (134 discarded)\n",
      "[2016-07-03 02:32:13] Building transcriptome data files /home/jovyan/data/th_out/7A_E/tmp/ecoli_w3110\n",
      "[2016-07-03 02:32:14] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 02:32:18] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:32:59] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:33:39] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 02:33:39] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:33:50] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 02:33:51] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 02:33:52] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 02:33:53] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 02:33:54] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:34:05] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 02:34:06] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 02:34:07] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 02:34:08] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 02:34:09] Retrieving sequences for splices\n",
      "[2016-07-03 02:34:10] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 02:34:10] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 02:34:11] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 02:34:11] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 02:34:12] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 02:34:12] Joining segment hits\n",
      "[2016-07-03 02:34:13] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 02:34:14] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 02:34:15] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 02:34:15] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 02:34:16] Joining segment hits\n",
      "[2016-07-03 02:34:18] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 02:37:14] A summary of the alignment counts can be found in /home/jovyan/data/th_out/7A_E/align_summary.txt\n",
      "[2016-07-03 02:37:14] Run complete: 00:05:52 elapsed\n",
      "ln: failed to create hard link ‘/home/jovyan/data/th_out/7A_E.bam’: File exists\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 02:42:01] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 02:42:01] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 02:42:01] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 02:42:01] Checking for reference FASTA file\n",
      "[2016-07-03 02:42:01] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 02:42:01] Reading known junctions from GTF file\n",
      "[2016-07-03 02:42:01] Preparing reads\n",
      "\t left reads: min. length=19, max. length=101, 1441364 kept reads (62 discarded)\n",
      "\tright reads: min. length=24, max. length=101, 1441310 kept reads (116 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 02:42:54] Building transcriptome data files /home/jovyan/data/th_out/7A_G/tmp/ecoli_w3110\n",
      "[2016-07-03 02:42:55] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 02:42:59] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:43:42] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:44:24] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 02:44:24] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:44:35] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 02:44:36] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 02:44:37] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 02:44:38] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 02:44:39] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:44:52] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 02:44:54] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 02:44:55] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 02:44:56] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 02:44:57] Retrieving sequences for splices\n",
      "[2016-07-03 02:44:57] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 02:44:58] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 02:44:59] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 02:44:59] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 02:44:59] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 02:45:00] Joining segment hits\n",
      "[2016-07-03 02:45:01] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 02:45:02] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 02:45:03] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 02:45:03] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 02:45:04] Joining segment hits\n",
      "[2016-07-03 02:45:06] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 02:48:09] A summary of the alignment counts can be found in /home/jovyan/data/th_out/7A_G/align_summary.txt\n",
      "[2016-07-03 02:48:09] Run complete: 00:06:08 elapsed\n",
      "ln: failed to create hard link ‘/home/jovyan/data/th_out/7A_G.bam’: File exists\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 02:53:05] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 02:53:05] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 02:53:05] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 02:53:05] Checking for reference FASTA file\n",
      "[2016-07-03 02:53:05] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 02:53:06] Reading known junctions from GTF file\n",
      "[2016-07-03 02:53:06] Preparing reads\n",
      "\t left reads: min. length=35, max. length=101, 1409017 kept reads (155 discarded)\n",
      "\tright reads: min. length=20, max. length=101, 1408948 kept reads (224 discarded)\n",
      "[2016-07-03 02:53:58] Building transcriptome data files /home/jovyan/data/th_out/7A_K/tmp/ecoli_w3110\n",
      "[2016-07-03 02:53:59] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 02:54:03] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:54:44] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:55:25] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 02:55:25] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:55:41] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 02:55:42] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 02:55:43] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 02:55:44] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 02:55:45] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 02:55:59] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 02:56:01] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 02:56:02] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 02:56:03] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 02:56:04] Retrieving sequences for splices\n",
      "[2016-07-03 02:56:04] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 02:56:05] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 02:56:05] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 02:56:06] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 02:56:06] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 02:56:06] Joining segment hits\n",
      "[2016-07-03 02:56:08] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 02:56:09] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 02:56:09] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 02:56:10] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 02:56:11] Joining segment hits\n",
      "[2016-07-03 02:56:13] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 02:59:08] A summary of the alignment counts can be found in /home/jovyan/data/th_out/7A_K/align_summary.txt\n",
      "[2016-07-03 02:59:08] Run complete: 00:06:03 elapsed\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 03:03:55] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:03:55] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 03:03:55] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 03:03:55] Checking for reference FASTA file\n",
      "[2016-07-03 03:03:55] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 03:03:56] Reading known junctions from GTF file\n",
      "[2016-07-03 03:03:56] Preparing reads\n",
      "\t left reads: min. length=22, max. length=101, 1732065 kept reads (42 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1731974 kept reads (133 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 03:05:01] Building transcriptome data files /home/jovyan/data/th_out/7A_N/tmp/ecoli_w3110\n",
      "[2016-07-03 03:05:01] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 03:05:06] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:05:59] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:06:50] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 03:06:50] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:07:02] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:07:04] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:07:06] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:07:07] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:07:09] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:07:22] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:07:24] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:07:25] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:07:27] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:07:29] Retrieving sequences for splices\n",
      "[2016-07-03 03:07:29] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 03:07:30] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:07:31] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:07:31] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:07:31] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:07:32] Joining segment hits\n",
      "[2016-07-03 03:07:35] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:07:35] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:07:36] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:07:37] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:07:37] Joining segment hits\n",
      "[2016-07-03 03:07:40] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/7A_N/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/7A_N/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/7A_N/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/7A_N/junctions.bed /home/jovyan/data/th_out/7A_N/insertions.bed /home/jovyan/data/th_out/7A_N/deletions.bed /home/jovyan/data/th_out/7A_N/fusions.out /home/jovyan/data/th_out/7A_N/tmp/accepted_hits /home/jovyan/data/th_out/7A_N/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/7A_N/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7A_N/tmp/left_kept_reads.bam /home/jovyan/data/th_out/7A_N/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/7A_N/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7A_N/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/7A_N/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/7A_N/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/7A_N/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 03:07:59] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:07:59] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 03:07:59] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 03:07:59] Checking for reference FASTA file\n",
      "[2016-07-03 03:07:59] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 03:08:00] Reading known junctions from GTF file\n",
      "[2016-07-03 03:08:00] Preparing reads\n",
      "\t left reads: min. length=21, max. length=101, 1377356 kept reads (31 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1377288 kept reads (99 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 03:08:52] Building transcriptome data files /home/jovyan/data/th_out/7A_P/tmp/ecoli_w3110\n",
      "[2016-07-03 03:08:52] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 03:08:57] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:09:37] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:10:18] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 03:10:18] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:10:29] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:10:30] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:10:31] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:10:32] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:10:34] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:10:46] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:10:47] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:10:49] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:10:50] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:10:51] Retrieving sequences for splices\n",
      "[2016-07-03 03:10:51] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 03:10:52] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:10:52] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:10:53] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:10:53] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:10:54] Joining segment hits\n",
      "[2016-07-03 03:10:55] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:10:56] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:10:57] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:10:57] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:10:58] Joining segment hits\n",
      "[2016-07-03 03:11:00] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:13:55] A summary of the alignment counts can be found in /home/jovyan/data/th_out/7A_P/align_summary.txt\n",
      "[2016-07-03 03:13:55] Run complete: 00:05:55 elapsed\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 03:18:34] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:18:34] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 03:18:34] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 03:18:34] Checking for reference FASTA file\n",
      "[2016-07-03 03:18:34] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 03:18:35] Reading known junctions from GTF file\n",
      "[2016-07-03 03:18:35] Preparing reads\n",
      "\t left reads: min. length=31, max. length=101, 1473798 kept reads (90 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1473728 kept reads (160 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 03:19:31] Building transcriptome data files /home/jovyan/data/th_out/7B_E/tmp/ecoli_w3110\n",
      "[2016-07-03 03:19:31] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 03:19:36] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:20:17] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:20:58] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 03:20:58] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:21:08] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:21:09] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:21:10] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:21:11] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:21:12] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:21:28] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:21:29] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:21:31] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:21:32] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:21:33] Retrieving sequences for splices\n",
      "[2016-07-03 03:21:34] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 03:21:34] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:21:35] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:21:35] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:21:36] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:21:36] Joining segment hits\n",
      "[2016-07-03 03:21:38] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:21:39] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:21:39] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:21:40] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:21:41] Joining segment hits\n",
      "[2016-07-03 03:21:43] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:24:49] A summary of the alignment counts can be found in /home/jovyan/data/th_out/7B_E/align_summary.txt\n",
      "[2016-07-03 03:24:49] Run complete: 00:06:15 elapsed\n",
      "ln: failed to create hard link ‘/home/jovyan/data/th_out/7B_E.bam’: File exists\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 03:29:48] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:29:48] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 03:29:49] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 03:29:49] Checking for reference FASTA file\n",
      "[2016-07-03 03:29:49] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 03:29:49] Reading known junctions from GTF file\n",
      "[2016-07-03 03:29:49] Preparing reads\n",
      "\t left reads: min. length=39, max. length=101, 1632738 kept reads (98 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1632681 kept reads (155 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 03:30:50] Building transcriptome data files /home/jovyan/data/th_out/7B_G/tmp/ecoli_w3110\n",
      "[2016-07-03 03:30:50] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 03:30:55] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:31:43] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:32:29] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 03:32:29] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:32:43] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:32:44] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:32:45] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:32:46] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:32:47] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:33:02] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:33:04] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:33:05] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:33:06] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:33:07] Retrieving sequences for splices\n",
      "[2016-07-03 03:33:08] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 03:33:08] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:33:09] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:33:09] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:33:10] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:33:10] Joining segment hits\n",
      "[2016-07-03 03:33:13] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:33:13] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:33:14] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:33:15] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:33:15] Joining segment hits\n",
      "[2016-07-03 03:33:17] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/7B_G/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/7B_G/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/7B_G/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/7B_G/junctions.bed /home/jovyan/data/th_out/7B_G/insertions.bed /home/jovyan/data/th_out/7B_G/deletions.bed /home/jovyan/data/th_out/7B_G/fusions.out /home/jovyan/data/th_out/7B_G/tmp/accepted_hits /home/jovyan/data/th_out/7B_G/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/7B_G/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7B_G/tmp/left_kept_reads.bam /home/jovyan/data/th_out/7B_G/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/7B_G/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7B_G/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/7B_G/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/7B_G/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/7B_G/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 03:33:37] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:33:37] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 03:33:37] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 03:33:37] Checking for reference FASTA file\n",
      "[2016-07-03 03:33:37] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 03:33:37] Reading known junctions from GTF file\n",
      "[2016-07-03 03:33:37] Preparing reads\n",
      "\t left reads: min. length=35, max. length=101, 1620641 kept reads (144 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1620541 kept reads (244 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 03:34:38] Building transcriptome data files /home/jovyan/data/th_out/7B_K/tmp/ecoli_w3110\n",
      "[2016-07-03 03:34:38] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 03:34:43] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:35:27] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:36:12] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 03:36:12] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:36:30] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:36:32] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:36:33] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:36:34] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:36:36] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:36:56] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:36:57] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:36:59] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:37:00] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:37:01] Retrieving sequences for splices\n",
      "[2016-07-03 03:37:01] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 03:37:02] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:37:03] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:37:03] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:37:04] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:37:04] Joining segment hits\n",
      "[2016-07-03 03:37:06] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:37:07] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:37:07] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:37:08] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:37:09] Joining segment hits\n",
      "[2016-07-03 03:37:11] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:40:31] A summary of the alignment counts can be found in /home/jovyan/data/th_out/7B_K/align_summary.txt\n",
      "[2016-07-03 03:40:31] Run complete: 00:06:53 elapsed\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 03:45:51] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:45:51] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 03:45:51] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 03:45:51] Checking for reference FASTA file\n",
      "[2016-07-03 03:45:51] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 03:45:52] Reading known junctions from GTF file\n",
      "[2016-07-03 03:45:52] Preparing reads\n",
      "\t left reads: min. length=20, max. length=101, 1059499 kept reads (59 discarded)\n",
      "\tright reads: min. length=21, max. length=101, 1059445 kept reads (113 discarded)\n",
      "[2016-07-03 03:46:31] Building transcriptome data files /home/jovyan/data/th_out/7B_N/tmp/ecoli_w3110\n",
      "[2016-07-03 03:46:31] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 03:46:36] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:47:15] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:47:51] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 03:47:51] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:48:02] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:48:03] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:48:05] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:48:06] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:48:08] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:48:20] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:48:22] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:48:24] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:48:25] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:48:27] Retrieving sequences for splices\n",
      "[2016-07-03 03:48:27] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 03:48:28] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:48:29] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:48:29] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:48:30] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:48:30] Joining segment hits\n",
      "[2016-07-03 03:48:32] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:48:33] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:48:33] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:48:34] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:48:34] Joining segment hits\n",
      "[2016-07-03 03:48:36] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:50:50] A summary of the alignment counts can be found in /home/jovyan/data/th_out/7B_N/align_summary.txt\n",
      "[2016-07-03 03:50:50] Run complete: 00:04:58 elapsed\n",
      "\n",
      "[2016-07-03 03:54:11] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 03:54:11] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 03:54:11] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 03:54:11] Checking for reference FASTA file\n",
      "[2016-07-03 03:54:11] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 03:54:12] Reading known junctions from GTF file\n",
      "[2016-07-03 03:54:12] Preparing reads\n",
      "\t left reads: min. length=23, max. length=101, 1593907 kept reads (202 discarded)\n",
      "\tright reads: min. length=21, max. length=101, 1593823 kept reads (286 discarded)\n",
      "[2016-07-03 03:55:10] Building transcriptome data files /home/jovyan/data/th_out/7B_P/tmp/ecoli_w3110\n",
      "[2016-07-03 03:55:10] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 03:55:15] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:55:59] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:56:42] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 03:56:42] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:56:54] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:56:55] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:56:56] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:56:57] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:56:59] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 03:57:15] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 03:57:16] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 03:57:18] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 03:57:19] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 03:57:20] Retrieving sequences for splices\n",
      "[2016-07-03 03:57:21] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 03:57:21] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:57:22] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:57:22] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:57:23] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:57:24] Joining segment hits\n",
      "[2016-07-03 03:57:25] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 03:57:26] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 03:57:27] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 03:57:28] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 03:57:28] Joining segment hits\n",
      "[2016-07-03 03:57:30] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:00:51] A summary of the alignment counts can be found in /home/jovyan/data/th_out/7B_P/align_summary.txt\n",
      "[2016-07-03 04:00:51] Run complete: 00:06:39 elapsed\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 04:06:17] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:06:17] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 04:06:17] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 04:06:17] Checking for reference FASTA file\n",
      "[2016-07-03 04:06:17] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 04:06:17] Reading known junctions from GTF file\n",
      "[2016-07-03 04:06:17] Preparing reads\n",
      "\t left reads: min. length=30, max. length=101, 1729990 kept reads (94 discarded)\n",
      "\tright reads: min. length=21, max. length=101, 1729929 kept reads (155 discarded)\n",
      "[2016-07-03 04:07:23] Building transcriptome data files /home/jovyan/data/th_out/7C_E/tmp/ecoli_w3110\n",
      "[2016-07-03 04:07:23] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 04:07:28] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:08:15] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:09:03] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 04:09:03] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:09:16] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:09:18] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:09:19] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:09:20] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:09:21] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:09:36] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:09:38] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:09:39] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:09:40] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:09:42] Retrieving sequences for splices\n",
      "[2016-07-03 04:09:42] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 04:09:43] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:09:43] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:09:44] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:09:44] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:09:45] Joining segment hits\n",
      "[2016-07-03 04:09:46] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:09:47] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:09:48] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:09:48] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:09:49] Joining segment hits\n",
      "[2016-07-03 04:09:51] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/7C_E/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/7C_E/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/7C_E/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/7C_E/junctions.bed /home/jovyan/data/th_out/7C_E/insertions.bed /home/jovyan/data/th_out/7C_E/deletions.bed /home/jovyan/data/th_out/7C_E/fusions.out /home/jovyan/data/th_out/7C_E/tmp/accepted_hits /home/jovyan/data/th_out/7C_E/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/7C_E/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7C_E/tmp/left_kept_reads.bam /home/jovyan/data/th_out/7C_E/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/7C_E/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7C_E/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/7C_E/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/7C_E/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/7C_E/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 04:10:12] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:10:12] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 04:10:12] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 04:10:12] Checking for reference FASTA file\n",
      "[2016-07-03 04:10:12] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 04:10:13] Reading known junctions from GTF file\n",
      "[2016-07-03 04:10:13] Preparing reads\n",
      "\t left reads: min. length=42, max. length=101, 1501394 kept reads (69 discarded)\n",
      "\tright reads: min. length=20, max. length=101, 1501350 kept reads (113 discarded)\n",
      "[2016-07-03 04:11:09] Building transcriptome data files /home/jovyan/data/th_out/7C_G/tmp/ecoli_w3110\n",
      "[2016-07-03 04:11:10] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 04:11:14] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:11:57] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:12:40] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 04:12:40] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:12:50] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:12:51] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:12:52] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:12:53] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:12:54] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:13:10] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:13:11] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:13:13] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:13:14] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:13:15] Retrieving sequences for splices\n",
      "[2016-07-03 04:13:15] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 04:13:16] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:13:16] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:13:17] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:13:17] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:13:17] Joining segment hits\n",
      "[2016-07-03 04:13:19] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:13:20] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:13:20] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:13:21] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:13:21] Joining segment hits\n",
      "[2016-07-03 04:13:24] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:16:38] A summary of the alignment counts can be found in /home/jovyan/data/th_out/7C_G/align_summary.txt\n",
      "[2016-07-03 04:16:38] Run complete: 00:06:25 elapsed\n",
      "ln: failed to create hard link ‘/home/jovyan/data/th_out/7C_G.bam’: File exists\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 04:21:42] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:21:42] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 04:21:42] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 04:21:42] Checking for reference FASTA file\n",
      "[2016-07-03 04:21:42] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 04:21:43] Reading known junctions from GTF file\n",
      "[2016-07-03 04:21:43] Preparing reads\n",
      "\t left reads: min. length=25, max. length=101, 1443086 kept reads (78 discarded)\n",
      "\tright reads: min. length=22, max. length=101, 1443015 kept reads (149 discarded)\n",
      "[2016-07-03 04:22:38] Building transcriptome data files /home/jovyan/data/th_out/7C_K/tmp/ecoli_w3110\n",
      "[2016-07-03 04:22:38] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 04:22:43] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:23:24] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:24:05] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 04:24:05] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:24:15] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:24:16] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:24:17] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:24:18] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:24:19] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:24:34] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:24:36] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:24:37] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:24:38] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:24:39] Retrieving sequences for splices\n",
      "[2016-07-03 04:24:39] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 04:24:40] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:24:40] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:24:41] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:24:41] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:24:42] Joining segment hits\n",
      "[2016-07-03 04:24:43] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:24:44] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:24:45] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:24:45] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:24:46] Joining segment hits\n",
      "[2016-07-03 04:24:48] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/7C_K/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/7C_K/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/7C_K/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/7C_K/junctions.bed /home/jovyan/data/th_out/7C_K/insertions.bed /home/jovyan/data/th_out/7C_K/deletions.bed /home/jovyan/data/th_out/7C_K/fusions.out /home/jovyan/data/th_out/7C_K/tmp/accepted_hits /home/jovyan/data/th_out/7C_K/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/7C_K/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7C_K/tmp/left_kept_reads.bam /home/jovyan/data/th_out/7C_K/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/7C_K/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7C_K/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/7C_K/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/7C_K/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/7C_K/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 04:25:05] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:25:05] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 04:25:05] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 04:25:05] Checking for reference FASTA file\n",
      "[2016-07-03 04:25:05] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 04:25:06] Reading known junctions from GTF file\n",
      "[2016-07-03 04:25:06] Preparing reads\n",
      "\t left reads: min. length=23, max. length=101, 1288216 kept reads (64 discarded)\n",
      "\tright reads: min. length=21, max. length=101, 1288157 kept reads (123 discarded)\n",
      "[2016-07-03 04:25:55] Building transcriptome data files /home/jovyan/data/th_out/7C_N/tmp/ecoli_w3110\n",
      "[2016-07-03 04:25:55] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 04:26:00] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:26:41] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:27:22] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 04:27:22] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:27:34] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:27:36] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:27:37] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:27:39] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:27:40] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:27:53] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:27:55] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:27:56] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:27:58] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:27:59] Retrieving sequences for splices\n",
      "[2016-07-03 04:28:00] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 04:28:00] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:28:01] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:28:01] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:28:02] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:28:02] Joining segment hits\n",
      "[2016-07-03 04:28:04] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:28:05] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:28:05] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:28:06] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:28:06] Joining segment hits\n",
      "[2016-07-03 04:28:09] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/7C_N/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/7C_N/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/7C_N/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/7C_N/junctions.bed /home/jovyan/data/th_out/7C_N/insertions.bed /home/jovyan/data/th_out/7C_N/deletions.bed /home/jovyan/data/th_out/7C_N/fusions.out /home/jovyan/data/th_out/7C_N/tmp/accepted_hits /home/jovyan/data/th_out/7C_N/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/7C_N/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7C_N/tmp/left_kept_reads.bam /home/jovyan/data/th_out/7C_N/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/7C_N/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7C_N/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/7C_N/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/7C_N/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/7C_N/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 04:28:27] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:28:27] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 04:28:27] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 04:28:27] Checking for reference FASTA file\n",
      "[2016-07-03 04:28:27] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 04:28:27] Reading known junctions from GTF file\n",
      "[2016-07-03 04:28:27] Preparing reads\n",
      "\t left reads: min. length=21, max. length=101, 1333767 kept reads (97 discarded)\n",
      "\tright reads: min. length=20, max. length=101, 1333686 kept reads (178 discarded)\n",
      "[2016-07-03 04:29:17] Building transcriptome data files /home/jovyan/data/th_out/7C_P/tmp/ecoli_w3110\n",
      "[2016-07-03 04:29:18] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 04:29:22] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:30:01] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:30:38] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 04:30:38] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:30:50] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:30:51] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:30:52] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:30:53] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:30:54] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:31:05] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:31:06] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:31:07] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:31:08] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:31:09] Retrieving sequences for splices\n",
      "[2016-07-03 04:31:10] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 04:31:10] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:31:11] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:31:12] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:31:12] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:31:13] Joining segment hits\n",
      "[2016-07-03 04:31:14] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:31:15] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:31:15] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:31:16] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:31:16] Joining segment hits\n",
      "[2016-07-03 04:31:18] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/7C_P/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/7C_P/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/7C_P/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/7C_P/junctions.bed /home/jovyan/data/th_out/7C_P/insertions.bed /home/jovyan/data/th_out/7C_P/deletions.bed /home/jovyan/data/th_out/7C_P/fusions.out /home/jovyan/data/th_out/7C_P/tmp/accepted_hits /home/jovyan/data/th_out/7C_P/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/7C_P/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7C_P/tmp/left_kept_reads.bam /home/jovyan/data/th_out/7C_P/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/7C_P/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/7C_P/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/7C_P/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/7C_P/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/7C_P/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 04:31:39] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:31:39] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 04:31:39] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 04:31:39] Checking for reference FASTA file\n",
      "[2016-07-03 04:31:39] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 04:31:40] Reading known junctions from GTF file\n",
      "[2016-07-03 04:31:40] Preparing reads\n",
      "\t left reads: min. length=33, max. length=101, 1741265 kept reads (76 discarded)\n",
      "\tright reads: min. length=22, max. length=101, 1741212 kept reads (129 discarded)\n",
      "[2016-07-03 04:32:42] Building transcriptome data files /home/jovyan/data/th_out/8A_E/tmp/ecoli_w3110\n",
      "[2016-07-03 04:32:43] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 04:32:47] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:33:35] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:34:23] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 04:34:23] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:34:37] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:34:38] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:34:40] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:34:41] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:34:42] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:35:01] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:35:02] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:35:03] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:35:05] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:35:06] Retrieving sequences for splices\n",
      "[2016-07-03 04:35:06] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 04:35:07] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:35:07] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:35:08] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:35:08] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:35:09] Joining segment hits\n",
      "[2016-07-03 04:35:10] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:35:11] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:35:12] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:35:12] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:35:13] Joining segment hits\n",
      "[2016-07-03 04:35:15] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:38:55] A summary of the alignment counts can be found in /home/jovyan/data/th_out/8A_E/align_summary.txt\n",
      "[2016-07-03 04:38:55] Run complete: 00:07:16 elapsed\n",
      "ln: failed to create hard link ‘/home/jovyan/data/th_out/8A_E.bam’: File exists\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 04:44:52] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:44:52] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 04:44:52] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 04:44:52] Checking for reference FASTA file\n",
      "[2016-07-03 04:44:52] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 04:44:52] Reading known junctions from GTF file\n",
      "[2016-07-03 04:44:53] Preparing reads\n",
      "\t left reads: min. length=36, max. length=101, 1853434 kept reads (57 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1853374 kept reads (117 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 04:46:01] Building transcriptome data files /home/jovyan/data/th_out/8A_G/tmp/ecoli_w3110\n",
      "[2016-07-03 04:46:01] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 04:46:06] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:46:58] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:47:50] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 04:47:50] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:48:03] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:48:05] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:48:06] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:48:07] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:48:08] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 04:48:26] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 04:48:28] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 04:48:29] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 04:48:31] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 04:48:32] Retrieving sequences for splices\n",
      "[2016-07-03 04:48:32] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 04:48:33] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:48:34] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:48:34] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:48:34] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:48:35] Joining segment hits\n",
      "[2016-07-03 04:48:37] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 04:48:37] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 04:48:38] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 04:48:39] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 04:48:39] Joining segment hits\n",
      "[2016-07-03 04:48:42] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:52:38] A summary of the alignment counts can be found in /home/jovyan/data/th_out/8A_G/align_summary.txt\n",
      "[2016-07-03 04:52:38] Run complete: 00:07:46 elapsed\n",
      "ln: failed to create hard link ‘/home/jovyan/data/th_out/8A_G.bam’: File exists\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 04:58:47] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 04:58:47] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 04:58:47] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 04:58:47] Checking for reference FASTA file\n",
      "[2016-07-03 04:58:47] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 04:58:47] Reading known junctions from GTF file\n",
      "[2016-07-03 04:58:47] Preparing reads\n",
      "\t left reads: min. length=43, max. length=101, 1189825 kept reads (6 discarded)\n",
      "\tright reads: min. length=20, max. length=101, 1189744 kept reads (87 discarded)\n",
      "[2016-07-03 04:59:31] Building transcriptome data files /home/jovyan/data/th_out/8A_K/tmp/ecoli_w3110\n",
      "[2016-07-03 04:59:32] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 04:59:36] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:00:12] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:00:47] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 05:00:47] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:00:57] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:00:58] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:01:00] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:01:01] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:01:02] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:01:12] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:01:13] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:01:14] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:01:15] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:01:16] Retrieving sequences for splices\n",
      "[2016-07-03 05:01:16] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 05:01:17] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:01:18] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:01:18] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:01:19] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:01:19] Joining segment hits\n",
      "[2016-07-03 05:01:20] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:01:21] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:01:21] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:01:22] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:01:23] Joining segment hits\n",
      "[2016-07-03 05:01:24] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/8A_K/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/8A_K/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/8A_K/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/8A_K/junctions.bed /home/jovyan/data/th_out/8A_K/insertions.bed /home/jovyan/data/th_out/8A_K/deletions.bed /home/jovyan/data/th_out/8A_K/fusions.out /home/jovyan/data/th_out/8A_K/tmp/accepted_hits /home/jovyan/data/th_out/8A_K/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/8A_K/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8A_K/tmp/left_kept_reads.bam /home/jovyan/data/th_out/8A_K/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/8A_K/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8A_K/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/8A_K/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/8A_K/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/8A_K/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 05:01:40] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:01:40] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 05:01:40] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 05:01:40] Checking for reference FASTA file\n",
      "[2016-07-03 05:01:40] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 05:01:41] Reading known junctions from GTF file\n",
      "[2016-07-03 05:01:41] Preparing reads\n",
      "\t left reads: min. length=20, max. length=101, 1156996 kept reads (58 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1156936 kept reads (118 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 05:02:24] Building transcriptome data files /home/jovyan/data/th_out/8A_N/tmp/ecoli_w3110\n",
      "[2016-07-03 05:02:24] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 05:02:29] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:03:07] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:03:44] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 05:03:44] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:03:55] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:03:57] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:03:59] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:04:01] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:04:02] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:04:19] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:04:21] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:04:22] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:04:24] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:04:26] Retrieving sequences for splices\n",
      "[2016-07-03 05:04:26] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 05:04:27] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:04:27] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:04:28] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:04:29] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:04:29] Joining segment hits\n",
      "[2016-07-03 05:04:31] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:04:31] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:04:32] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:04:33] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:04:34] Joining segment hits\n",
      "[2016-07-03 05:04:36] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/8A_N/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/8A_N/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/8A_N/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/8A_N/junctions.bed /home/jovyan/data/th_out/8A_N/insertions.bed /home/jovyan/data/th_out/8A_N/deletions.bed /home/jovyan/data/th_out/8A_N/fusions.out /home/jovyan/data/th_out/8A_N/tmp/accepted_hits /home/jovyan/data/th_out/8A_N/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/8A_N/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8A_N/tmp/left_kept_reads.bam /home/jovyan/data/th_out/8A_N/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/8A_N/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8A_N/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/8A_N/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/8A_N/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/8A_N/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 05:04:51] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:04:51] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 05:04:51] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 05:04:51] Checking for reference FASTA file\n",
      "[2016-07-03 05:04:51] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 05:04:52] Reading known junctions from GTF file\n",
      "[2016-07-03 05:04:52] Preparing reads\n",
      "\t left reads: min. length=37, max. length=101, 1144588 kept reads (50 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1144531 kept reads (107 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 05:05:34] Building transcriptome data files /home/jovyan/data/th_out/8A_P/tmp/ecoli_w3110\n",
      "[2016-07-03 05:05:34] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 05:05:39] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:06:13] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:06:48] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 05:06:48] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:06:55] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:06:56] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:06:57] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:06:58] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:06:59] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:07:09] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:07:10] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:07:11] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:07:12] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:07:13] Retrieving sequences for splices\n",
      "[2016-07-03 05:07:13] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 05:07:14] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:07:14] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:07:14] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:07:15] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:07:15] Joining segment hits\n",
      "[2016-07-03 05:07:17] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:07:17] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:07:18] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:07:18] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:07:19] Joining segment hits\n",
      "[2016-07-03 05:07:20] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:09:42] A summary of the alignment counts can be found in /home/jovyan/data/th_out/8A_P/align_summary.txt\n",
      "[2016-07-03 05:09:42] Run complete: 00:04:50 elapsed\n",
      "\n",
      "[2016-07-03 05:13:26] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:13:26] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 05:13:26] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 05:13:26] Checking for reference FASTA file\n",
      "[2016-07-03 05:13:26] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 05:13:26] Reading known junctions from GTF file\n",
      "[2016-07-03 05:13:27] Preparing reads\n",
      "\t left reads: min. length=29, max. length=101, 1855838 kept reads (97 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1855771 kept reads (164 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 05:14:35] Building transcriptome data files /home/jovyan/data/th_out/8B_E/tmp/ecoli_w3110\n",
      "[2016-07-03 05:14:35] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 05:14:40] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:15:30] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:16:20] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 05:16:20] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:16:34] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:16:35] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:16:36] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:16:37] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:16:38] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:16:59] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:17:00] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:17:02] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:17:03] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:17:05] Retrieving sequences for splices\n",
      "[2016-07-03 05:17:05] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 05:17:06] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:17:06] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:17:07] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:17:07] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:17:08] Joining segment hits\n",
      "[2016-07-03 05:17:10] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:17:11] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:17:11] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:17:12] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:17:13] Joining segment hits\n",
      "[2016-07-03 05:17:15] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:21:11] A summary of the alignment counts can be found in /home/jovyan/data/th_out/8B_E/align_summary.txt\n",
      "[2016-07-03 05:21:11] Run complete: 00:07:44 elapsed\n",
      "ln: failed to create hard link ‘/home/jovyan/data/th_out/8B_E.bam’: File exists\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 05:27:24] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:27:24] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 05:27:24] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 05:27:24] Checking for reference FASTA file\n",
      "[2016-07-03 05:27:24] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 05:27:25] Reading known junctions from GTF file\n",
      "[2016-07-03 05:27:25] Preparing reads\n",
      "\t left reads: min. length=45, max. length=101, 1245716 kept reads (91 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1245658 kept reads (149 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 05:28:12] Building transcriptome data files /home/jovyan/data/th_out/8B_G/tmp/ecoli_w3110\n",
      "[2016-07-03 05:28:12] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 05:28:17] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:28:55] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:30:04] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 05:30:04] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:30:13] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:30:15] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:30:16] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:30:17] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:30:19] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:30:33] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:30:35] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:30:36] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:30:38] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:30:39] Retrieving sequences for splices\n",
      "[2016-07-03 05:30:39] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 05:30:40] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:30:41] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:30:41] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:30:42] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:30:42] Joining segment hits\n",
      "[2016-07-03 05:30:44] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:30:47] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:30:48] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:30:50] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:30:51] Joining segment hits\n",
      "[2016-07-03 05:30:54] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/8B_G/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/8B_G/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/8B_G/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/8B_G/junctions.bed /home/jovyan/data/th_out/8B_G/insertions.bed /home/jovyan/data/th_out/8B_G/deletions.bed /home/jovyan/data/th_out/8B_G/fusions.out /home/jovyan/data/th_out/8B_G/tmp/accepted_hits /home/jovyan/data/th_out/8B_G/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/8B_G/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8B_G/tmp/left_kept_reads.bam /home/jovyan/data/th_out/8B_G/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/8B_G/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8B_G/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/8B_G/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/8B_G/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/8B_G/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 05:31:15] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:31:15] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 05:31:15] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 05:31:15] Checking for reference FASTA file\n",
      "[2016-07-03 05:31:15] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 05:31:15] Reading known junctions from GTF file\n",
      "[2016-07-03 05:31:16] Preparing reads\n",
      "\t left reads: min. length=41, max. length=101, 1602477 kept reads (44 discarded)\n",
      "\tright reads: min. length=21, max. length=101, 1602378 kept reads (143 discarded)\n",
      "[2016-07-03 05:32:13] Building transcriptome data files /home/jovyan/data/th_out/8B_K/tmp/ecoli_w3110\n",
      "[2016-07-03 05:32:14] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 05:32:19] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:33:03] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:33:47] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 05:33:47] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:34:00] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:34:01] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:34:02] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:34:03] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:34:04] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:34:17] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:34:18] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:34:19] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:34:20] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:34:21] Retrieving sequences for splices\n",
      "[2016-07-03 05:34:22] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 05:34:22] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:34:23] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:34:23] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:34:24] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:34:24] Joining segment hits\n",
      "[2016-07-03 05:34:26] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:34:27] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:34:28] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:34:28] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:34:29] Joining segment hits\n",
      "[2016-07-03 05:34:31] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/8B_K/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/8B_K/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/8B_K/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/8B_K/junctions.bed /home/jovyan/data/th_out/8B_K/insertions.bed /home/jovyan/data/th_out/8B_K/deletions.bed /home/jovyan/data/th_out/8B_K/fusions.out /home/jovyan/data/th_out/8B_K/tmp/accepted_hits /home/jovyan/data/th_out/8B_K/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/8B_K/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8B_K/tmp/left_kept_reads.bam /home/jovyan/data/th_out/8B_K/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/8B_K/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8B_K/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/8B_K/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/8B_K/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/8B_K/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 05:34:46] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:34:46] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 05:34:46] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 05:34:46] Checking for reference FASTA file\n",
      "[2016-07-03 05:34:46] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 05:34:46] Reading known junctions from GTF file\n",
      "[2016-07-03 05:34:46] Preparing reads\n",
      "\t left reads: min. length=34, max. length=101, 954971 kept reads (331 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 954912 kept reads (390 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 05:35:22] Building transcriptome data files /home/jovyan/data/th_out/8B_N/tmp/ecoli_w3110\n",
      "[2016-07-03 05:35:22] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 05:35:27] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:36:00] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:36:32] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 05:36:32] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:36:45] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:36:47] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:36:48] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:36:50] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:36:51] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:37:06] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:37:08] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:37:10] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:37:11] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:37:13] Retrieving sequences for splices\n",
      "[2016-07-03 05:37:13] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 05:37:14] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:37:14] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:37:15] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:37:15] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:37:16] Joining segment hits\n",
      "[2016-07-03 05:37:17] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:37:18] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:37:19] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:37:20] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:37:20] Joining segment hits\n",
      "[2016-07-03 05:37:22] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:39:09] A summary of the alignment counts can be found in /home/jovyan/data/th_out/8B_N/align_summary.txt\n",
      "[2016-07-03 05:39:09] Run complete: 00:04:23 elapsed\n",
      "\n",
      "[2016-07-03 05:42:07] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:42:07] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 05:42:07] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 05:42:07] Checking for reference FASTA file\n",
      "[2016-07-03 05:42:07] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 05:42:08] Reading known junctions from GTF file\n",
      "[2016-07-03 05:42:08] Preparing reads\n",
      "\t left reads: min. length=26, max. length=101, 1196365 kept reads (70 discarded)\n",
      "\tright reads: min. length=23, max. length=101, 1196327 kept reads (108 discarded)\n",
      "[2016-07-03 05:42:52] Building transcriptome data files /home/jovyan/data/th_out/8B_P/tmp/ecoli_w3110\n",
      "[2016-07-03 05:42:52] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 05:42:57] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:43:32] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:44:08] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 05:44:08] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:44:16] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:44:17] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:44:18] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:44:19] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:44:20] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:44:30] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:44:31] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:44:33] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:44:34] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:44:35] Retrieving sequences for splices\n",
      "[2016-07-03 05:44:35] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 05:44:36] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:44:36] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:44:37] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:44:37] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:44:38] Joining segment hits\n",
      "[2016-07-03 05:44:39] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:44:39] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:44:40] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:44:40] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:44:41] Joining segment hits\n",
      "[2016-07-03 05:44:43] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/8B_P/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/8B_P/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/8B_P/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/8B_P/junctions.bed /home/jovyan/data/th_out/8B_P/insertions.bed /home/jovyan/data/th_out/8B_P/deletions.bed /home/jovyan/data/th_out/8B_P/fusions.out /home/jovyan/data/th_out/8B_P/tmp/accepted_hits /home/jovyan/data/th_out/8B_P/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/8B_P/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8B_P/tmp/left_kept_reads.bam /home/jovyan/data/th_out/8B_P/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/8B_P/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8B_P/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/8B_P/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/8B_P/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/8B_P/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 05:45:08] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:45:08] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 05:45:08] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 05:45:08] Checking for reference FASTA file\n",
      "[2016-07-03 05:45:08] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 05:45:09] Reading known junctions from GTF file\n",
      "[2016-07-03 05:45:09] Preparing reads\n",
      "\t left reads: min. length=35, max. length=101, 2072978 kept reads (81 discarded)\n",
      "\tright reads: min. length=20, max. length=101, 2072880 kept reads (179 discarded)\n",
      "[2016-07-03 05:46:27] Building transcriptome data files /home/jovyan/data/th_out/8C_E/tmp/ecoli_w3110\n",
      "[2016-07-03 05:46:27] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 05:46:33] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:47:27] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:48:22] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 05:48:22] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:48:42] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:48:43] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:48:44] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:48:45] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:48:47] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 05:49:09] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 05:49:11] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 05:49:12] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 05:49:14] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 05:49:15] Retrieving sequences for splices\n",
      "[2016-07-03 05:49:15] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 05:49:16] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:49:17] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:49:17] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:49:18] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:49:18] Joining segment hits\n",
      "[2016-07-03 05:49:20] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 05:49:21] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 05:49:21] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 05:49:23] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 05:49:23] Joining segment hits\n",
      "[2016-07-03 05:49:26] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 05:53:47] A summary of the alignment counts can be found in /home/jovyan/data/th_out/8C_E/align_summary.txt\n",
      "[2016-07-03 05:53:47] Run complete: 00:08:38 elapsed\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 06:00:45] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 06:00:45] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 06:00:45] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 06:00:45] Checking for reference FASTA file\n",
      "[2016-07-03 06:00:45] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 06:00:45] Reading known junctions from GTF file\n",
      "[2016-07-03 06:00:45] Preparing reads\n",
      "\t left reads: min. length=38, max. length=101, 1332602 kept reads (77 discarded)\n",
      "\tright reads: min. length=20, max. length=101, 1332540 kept reads (139 discarded)\n",
      "[2016-07-03 06:01:35] Building transcriptome data files /home/jovyan/data/th_out/8C_G/tmp/ecoli_w3110\n",
      "[2016-07-03 06:01:36] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 06:01:41] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:02:20] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:02:59] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 06:02:59] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:03:09] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 06:03:10] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 06:03:11] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 06:03:12] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 06:03:13] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:03:25] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 06:03:26] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 06:03:28] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 06:03:29] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 06:03:30] Retrieving sequences for splices\n",
      "[2016-07-03 06:03:30] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 06:03:30] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 06:03:31] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 06:03:31] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 06:03:32] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 06:03:32] Joining segment hits\n",
      "[2016-07-03 06:03:34] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 06:03:34] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 06:03:35] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 06:03:36] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 06:03:36] Joining segment hits\n",
      "[2016-07-03 06:03:38] Reporting output tracks\n",
      "\t[FAILED]\n",
      "Error running /usr/bin/tophat_reports --min-anchor 8 --splice-mismatches 0 --min-report-intron 4 --max-report-intron 5 --min-isoform-fraction 0.15 --output-dir /home/jovyan/data/th_out/8C_G/ --max-multihits 1 --max-seg-multihits 10 --segment-length 25 --segment-mismatches 2 --min-closure-exon 100 --min-closure-intron 50 --max-closure-intron 5 --min-coverage-intron 50 --max-coverage-intron 5 --min-segment-intron 4 --max-segment-intron 5 --read-mismatches 2 --read-gap-length 2 --read-edit-dist 2 --read-realign-edit-dist 3 --max-insertion-length 3 --max-deletion-length 3 -z gzip -p2 --inner-dist-mean 50 --inner-dist-std-dev 20 --gtf-annotations /home/jovyan/data/genome/ecoli_w3110.gff --gtf-juncs /home/jovyan/data/th_out/8C_G/tmp/ecoli_w3110.juncs --no-closure-search --no-coverage-search --no-microexon-search --library-type fr-firststrand --sam-header /home/jovyan/data/th_out/8C_G/tmp/ecoli_w3110_genome.bwt.samheader.sam --report-discordant-pair-alignments --report-mixed-alignments --samtools=/usr/bin/samtools_0.1.18 --bowtie2-max-penalty 6 --bowtie2-min-penalty 2 --bowtie2-penalty-for-N 1 --bowtie2-read-gap-open 5 --bowtie2-read-gap-cont 3 --bowtie2-ref-gap-open 5 --bowtie2-ref-gap-cont 3 /home/jovyan/data/genome/ecoli_w3110.fa /home/jovyan/data/th_out/8C_G/junctions.bed /home/jovyan/data/th_out/8C_G/insertions.bed /home/jovyan/data/th_out/8C_G/deletions.bed /home/jovyan/data/th_out/8C_G/fusions.out /home/jovyan/data/th_out/8C_G/tmp/accepted_hits /home/jovyan/data/th_out/8C_G/tmp/left_kept_reads.m2g.bam,/home/jovyan/data/th_out/8C_G/tmp/left_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8C_G/tmp/left_kept_reads.bam /home/jovyan/data/th_out/8C_G/tmp/right_kept_reads.m2g.bam,/home/jovyan/data/th_out/8C_G/tmp/right_kept_reads.m2g_um.mapped.bam /home/jovyan/data/th_out/8C_G/tmp/right_kept_reads.bam\n",
      "Warning: no input BAM records found.\n",
      "\n",
      "ln: failed to access ‘/home/jovyan/data/th_out/8C_G/accepted_hits.bam’: No such file or directory\n",
      "open: No such file or directory\n",
      "[bam_index_build2] fail to open the BAM file.\n",
      "open: No such file or directory\n",
      "[bam_sort_core] fail to open file /home/jovyan/data/th_out/8C_G/accepted_hits.bam\n",
      "  [Errno 2] No such file or directory: '/home/jovyan/data/th_out/8C_G/accepted_hits.name.bam'\n",
      "  [Exception type: IOError, raised in count.py:42]\n",
      "\n",
      "[2016-07-03 06:03:56] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 06:03:56] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 06:03:56] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 06:03:56] Checking for reference FASTA file\n",
      "[2016-07-03 06:03:56] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 06:03:56] Reading known junctions from GTF file\n",
      "[2016-07-03 06:03:56] Preparing reads\n",
      "\t left reads: min. length=19, max. length=101, 1303015 kept reads (48 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1302955 kept reads (108 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 06:04:45] Building transcriptome data files /home/jovyan/data/th_out/8C_K/tmp/ecoli_w3110\n",
      "[2016-07-03 06:04:45] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 06:04:50] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:05:28] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:06:06] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 06:06:06] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:06:16] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 06:06:17] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 06:06:19] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 06:06:20] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 06:06:22] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:06:35] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 06:06:36] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 06:06:38] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 06:06:39] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 06:06:40] Retrieving sequences for splices\n",
      "[2016-07-03 06:06:40] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 06:06:41] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 06:06:42] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 06:06:43] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 06:06:43] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 06:06:44] Joining segment hits\n",
      "[2016-07-03 06:06:45] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 06:06:46] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 06:06:47] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 06:06:48] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 06:06:48] Joining segment hits\n",
      "[2016-07-03 06:06:50] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 06:09:41] A summary of the alignment counts can be found in /home/jovyan/data/th_out/8C_K/align_summary.txt\n",
      "[2016-07-03 06:09:41] Run complete: 00:05:44 elapsed\n",
      "[bam_sort_core] merging from 2 files...\n",
      "\n",
      "[2016-07-03 06:14:20] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 06:14:20] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 06:14:20] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 06:14:20] Checking for reference FASTA file\n",
      "[2016-07-03 06:14:20] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 06:14:21] Reading known junctions from GTF file\n",
      "[2016-07-03 06:14:21] Preparing reads\n",
      "\t left reads: min. length=44, max. length=101, 991731 kept reads (933 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 991724 kept reads (940 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 06:14:58] Building transcriptome data files /home/jovyan/data/th_out/8C_N/tmp/ecoli_w3110\n",
      "[2016-07-03 06:14:59] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 06:15:04] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:15:36] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:16:09] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 06:16:09] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:16:20] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 06:16:22] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 06:16:24] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 06:16:25] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 06:16:26] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:16:40] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 06:16:41] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 06:16:43] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 06:16:44] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 06:16:46] Retrieving sequences for splices\n",
      "[2016-07-03 06:16:46] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 06:16:47] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 06:16:48] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 06:16:48] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 06:16:49] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 06:16:49] Joining segment hits\n",
      "[2016-07-03 06:16:51] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 06:16:52] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 06:16:53] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 06:16:54] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 06:16:54] Joining segment hits\n",
      "[2016-07-03 06:16:56] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 06:19:02] A summary of the alignment counts can be found in /home/jovyan/data/th_out/8C_N/align_summary.txt\n",
      "[2016-07-03 06:19:02] Run complete: 00:04:41 elapsed\n",
      "\n",
      "[2016-07-03 06:22:14] Beginning TopHat run (v2.0.13)\n",
      "-----------------------------------------------\n",
      "[2016-07-03 06:22:14] Checking for Bowtie\n",
      "\t\t  Bowtie version:\t 2.2.4.0\n",
      "[2016-07-03 06:22:14] Checking for Bowtie index files (genome)..\n",
      "[2016-07-03 06:22:14] Checking for reference FASTA file\n",
      "[2016-07-03 06:22:14] Generating SAM header for /home/jovyan/data/genome/ecoli_w3110\n",
      "[2016-07-03 06:22:15] Reading known junctions from GTF file\n",
      "[2016-07-03 06:22:15] Preparing reads\n",
      "\t left reads: min. length=38, max. length=101, 1281174 kept reads (113 discarded)\n",
      "\tright reads: min. length=19, max. length=101, 1281119 kept reads (168 discarded)\n",
      "Warning: short reads (<20bp) will make TopHat quite slow and take large amount of memory because they are likely to be mapped in too many places\n",
      "[2016-07-03 06:23:01] Building transcriptome data files /home/jovyan/data/th_out/8C_P/tmp/ecoli_w3110\n",
      "[2016-07-03 06:23:02] Building Bowtie index from ecoli_w3110.fa\n",
      "[2016-07-03 06:23:06] Mapping left_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:23:44] Mapping right_kept_reads to transcriptome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:24:21] Resuming TopHat pipeline with unmapped reads\n",
      "[2016-07-03 06:24:21] Mapping left_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:24:33] Mapping left_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 06:24:34] Mapping left_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 06:24:35] Mapping left_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 06:24:37] Mapping left_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 06:24:38] Mapping right_kept_reads.m2g_um to genome ecoli_w3110 with Bowtie2 \n",
      "[2016-07-03 06:24:51] Mapping right_kept_reads.m2g_um_seg1 to genome ecoli_w3110 with Bowtie2 (1/4)\n",
      "[2016-07-03 06:24:52] Mapping right_kept_reads.m2g_um_seg2 to genome ecoli_w3110 with Bowtie2 (2/4)\n",
      "[2016-07-03 06:24:53] Mapping right_kept_reads.m2g_um_seg3 to genome ecoli_w3110 with Bowtie2 (3/4)\n",
      "[2016-07-03 06:24:54] Mapping right_kept_reads.m2g_um_seg4 to genome ecoli_w3110 with Bowtie2 (4/4)\n",
      "[2016-07-03 06:24:56] Retrieving sequences for splices\n",
      "[2016-07-03 06:24:56] Indexing splices\n",
      "Building a SMALL index\n",
      "[2016-07-03 06:24:57] Mapping left_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 06:24:57] Mapping left_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 06:24:58] Mapping left_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 06:24:58] Mapping left_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 06:24:59] Joining segment hits\n",
      "[2016-07-03 06:25:00] Mapping right_kept_reads.m2g_um_seg1 to genome segment_juncs with Bowtie2 (1/4)\n",
      "[2016-07-03 06:25:01] Mapping right_kept_reads.m2g_um_seg2 to genome segment_juncs with Bowtie2 (2/4)\n",
      "[2016-07-03 06:25:02] Mapping right_kept_reads.m2g_um_seg3 to genome segment_juncs with Bowtie2 (3/4)\n",
      "[2016-07-03 06:25:02] Mapping right_kept_reads.m2g_um_seg4 to genome segment_juncs with Bowtie2 (4/4)\n",
      "[2016-07-03 06:25:03] Joining segment hits\n",
      "[2016-07-03 06:25:05] Reporting output tracks\n",
      "-----------------------------------------------\n",
      "[2016-07-03 06:27:49] A summary of the alignment counts can be found in /home/jovyan/data/th_out/8C_P/align_summary.txt\n",
      "[2016-07-03 06:27:49] Run complete: 00:05:35 elapsed\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mkdir -p ~/data/trimmed\n",
    "mkdir -p  ~/data/th_out\n",
    "mkdir -p  ~/data/counts\n",
    "\n",
    "FASTQ_DIR=~/data/fastqs\n",
    "\n",
    "IFS=\".\"\n",
    "# for R1 in ${FASTQ_DIR}/r1.7A_E.fq.gz\n",
    "for R1 in ${FASTQ_DIR}/r1.*.fq.gz\n",
    "    do\n",
    "        set $R1\n",
    "        BARCODE=$2\n",
    "        echo $BARCODE\n",
    "        fastq-mcf all_neb_adapters.fasta \\\n",
    "            ${FASTQ_DIR}/r1.${BARCODE}.fq.gz \\\n",
    "            ${FASTQ_DIR}/r2.${BARCODE}.fq.gz \\\n",
    "            -q 20 -x 0.5 \\\n",
    "            -o ~/data/trimmed/r1.${BARCODE}.trim.fq.gz \\\n",
    "            -o ~/data/trimmed/r2.${BARCODE}.trim.fq.gz\n",
    "            \n",
    "        mkdir -p ~/data/th_out/${BARCODE}\n",
    "        tophat2 -G ~/data/genome/ecoli_w3110.gff \\\n",
    "            --library-type fr-firststrand \\\n",
    "            --output-dir ~/data/th_out/${BARCODE} \\\n",
    "            --max-intron-length 5 \\\n",
    "            --min-intron-length 4 \\\n",
    "            --transcriptome-max-hits 1 \\\n",
    "            --max-multihits 1 \\\n",
    "            --no-coverage-search \\\n",
    "            --no-novel-juncs \\\n",
    "            --num-threads 2 \\\n",
    "            ~/data/genome/ecoli_w3110 \\\n",
    "            ~/data/trimmed/r1.${BARCODE}.trim.fq.gz \\\n",
    "            ~/data/trimmed/r2.${BARCODE}.trim.fq.gz\n",
    "            \n",
    "        ln ~/data/th_out/${BARCODE}/accepted_hits.bam ~/data/th_out/${BARCODE}.bam\n",
    "        samtools index ~/data/th_out/${BARCODE}.bam\n",
    "        \n",
    "        samtools sort -n ~/data/th_out/${BARCODE}/accepted_hits.bam \\\n",
    "            ~/data/th_out/${BARCODE}/accepted_hits.name\n",
    "\n",
    "        htseq-count --quiet --order=name --format=bam --stranded=reverse --type=gene \\\n",
    "            --idattr=ID ~/data/th_out/${BARCODE}/accepted_hits.name.bam \\\n",
    "            ~/data/genome/ecoli_w3110.gff > ~/data/counts/${BARCODE}.csv\n",
    "    done\n",
    "unset IFS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.11"
  },
  "widgets": {
   "state": {},
   "version": "1.1.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
